org.eclipse.scanning.malcolm.core/src/org/eclipse/scanning/malcolm/core/MalcolmDevice.java

```diff
@@ -69,6 +69,7 @@
 	private static String STATUS_ENDPOINT = "status";
 	private static String BUSY_ENDPOINT = "busy";
 	private static String CURRENT_STEP_ENDPOINT = "completedSteps";
+	private static String FILE_EXTENSION_H5 = "h5";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "FILE_EXTENSION_H5" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)